platform/vs/docker-dash-engine/start.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+simple_switch_grpc --interface 0@eth1 --interface 1@eth2 --log-console --no-p4
```

Comment:
I believe you mean the CPU port will be added in the future by @jimmyzhai , am I right?